crates/bitwarden-vault/src/collection.rs

```diff
@@ -85,3 +86,22 @@ impl TryFrom<CollectionDetailsResponseModel> for Collection {
         })
     }
 }
+
+impl TreeItem for CollectionView {
+    fn id(&self) -> Option<Uuid> {
+        self.id
+    }
+
+    fn short_name(&self) -> &str {
+        &self.path().last().unwrap()
+    }
```

Comment:
issue: unwrap is discouraged/not allowed, and while we allow `expect` it should be reserved for the very inlikely "This should never happen" scenarios.